pkg/handler/handler.go

```diff
@@ -240,22 +268,19 @@ func Handle(req []byte) string {
 	}
 
 	ctx, cancel := context.WithTimeout(context.Background(), cancelTimeout)
+
 	defer cancel()
-	var dataReader io.ReadCloser
-	dataReader, err = fetchReadmes(ctx, url)
-	if err != nil {
-		return renderError(err)
-	}
-	defer dataReader.Close()
 
-	readmes, err = parseReadmes(dataReader)
+	client, err := storage.NewClient(ctx, option.WithoutAuthentication())
 	if err != nil {
-		return renderError(err)
+		log.Fatalf(err.Error())
```

Comment:
:boom:

Reply:
I think openfaas concats return value, stdout and maybe stderr into the result, so this _may_ result in some fancy output. Not 110% sure tho

Reply:
ahh, I guess we can return the error. You prob want valid JSON whatever happens.